### PR TITLE
Changed the coding progress navbar and the layout of the body to work…

### PR DESCRIPTION
**Pull request by DanielMwendwa**
coding_progress
… with bootstrap

### CODE REVIEW

File: public/coda/styles.css [hunk 1]
```diff
@@ -1,53 +1,61 @@
...
 a {
     color: black;
 }
+.btn-logout:hover {
+    color: steelblue;
+    transform: scale(1.05);
+}
 .table-heading {
     background-color: #8e3436;
-    height: 30px;
```
Comment (IsaackMwenda) [inline public/coda/styles.css:19]: I think we need to control the heading height

File: public/coda/styles.css [hunk 2]
```diff
@@ -1,53 +1,61 @@
...
-    line-height:4px;
 }
 .coding-above75{
     background-color:#d6ffd9;
-    font-size:16;
-    line-height:3px;
 }
 .coding-complete{
     background-color: white;
-    font-size:15;
```
Comment (IsaackMwenda) [inline public/coda/styles.css:49]: ditto

File: public/coda/styles.css [hunk 3]
```diff
@@ -1,53 +1,61 @@
...
     background-color: #8e3436;
-    height: 30px;
     color:#ffff;
 }
+table > tbody > tr > td{
+    border-bottom: 1px solid rgb(0,0,0, 0.2);
+}
 .coding-notstarted{
     background-color:#fc9f9f;
-    font-size:16;
```
Comment (IsaackMwenda) [inline public/coda/styles.css:24]: ditto

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

File: public/coda/styles.css (hunk 2 above)
Comment (DanielMwendwa) [reply public/coda/styles.css:49]: Done

File: public/coda/styles.css [hunk 4]
```diff
@@ -1,53 +1,78 @@
...
+    border-bottom: 2px solid #8e3436;
+}
+table > tbody > tr > td{
+    border-bottom: 1px solid rgb(0,0,0, 0.2);
+}
 .coding-notstarted{
     background-color:#fc9f9f;
-    font-size:16;
-    line-height:4px;
+    font-size: 16;
```
Comment (IsaackMwenda) [inline public/coda/styles.css:49]: please be consistent with the var spacing

File: public/coda/styles.css [hunk 5]
```diff
@@ -83,18 +108,3 @@ a {
     text-align: center;
     font-size: 20px;
 }
-.container {
```
Comment (IsaackMwenda) [inline public/coda/styles.css:86]: As discussed offline please variate the container width and margins for different screen sizes. It feels a bit compressed in extra-large screens and the graphs overlay the operator legend.

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
please fix merge conflicts

**Comment by IsaackMwenda** [comment]
@DanielMwendwa whats the progress on this?

File: public/coda/styles.css (hunk 4 above)
Comment (DanielMwendwa) [reply public/coda/styles.css:49]: Okay

File: public/coda/styles.css (hunk 5 above)
Comment (DanielMwendwa) [reply public/coda/styles.css:86]: you can now look at the container width on extra-large screens

File: public/coda/styles.css [hunk 6]
```diff
@@ -83,18 +108,20 @@ a {
...
-  }
-  @media only screen and (min-width: 993px) {
+}
+@media screen and (min-width: 1900px) {
     .container {
-      width: 80%;
+        width: 80%;
     }
-  }
+}
```
Comment (IsaackMwenda) [inline public/coda/styles.css:127]: Please add a new line at the end of the file.

**Review by IsaackMwenda — APPROVED** [review]
Thanks, @DanielMwendwa. WorldBank-PLR_recently_displaced text is overflowing. We might need to increase the Dataset row width but in another pr.

**Comment by DanielMwendwa** [comment]
I had in mind to break this pr into smaller ones but since the changes are dependent on one another I have resolved to leave it like this.

**Pull request merged**